Week2/homework/js-exercises/LemonAllergy/lemonAllergy.js

```diff
@@ -0,0 +1,7 @@
+const fruitBasket = ['Apple', 'Lemon', 'Grapefruit', 'Lemon', 'Banana', 'Watermelon', 'Lemon'];
+
+const removeLemons = basket => {
+    return basket.filter(item => item !== 'Lemon')
```

Comment:
You can name the `item` however you like. This gives you the ability to say for example:
`basket.filter(fruit => fruit !== 'Lemon')`
This makes your code more readable